cli/index.js

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env node
+
+'use strict';
+
+var CryptoJS = require("crypto-js");
+var FileSystem = require("fs");
+
+const SCRIPT_URL = 'https://cdnjs.cloudflare.com/ajax/libs/crypto-js/3.1.9-1/crypto-js.min.js';
+const SCRIPT_TAG = '<script src="' + SCRIPT_URL + '" integrity="sha384-lp4k1VRKPU9eBnPePjnJ9M2RF3i7PC30gXs70+elCVfgwLwx1tv5+ctxdtwxqZa7" crossorigin="anonymous"></script>';
+
+const namedArgs = parseArgs(process.argv, process.argv.length);
+
+try{
+    var contents = FileSystem.readFileSync(namedArgs.input, 'utf8');
+}catch(e){
+    console.log("Failure: input file does not exist!");
+    process.exit(1);
+}
+
+var encrypted = CryptoJS.AES.encrypt(contents, namedArgs.password);
+var hmac = CryptoJS.HmacSHA256(encrypted.toString(), CryptoJS.SHA256(namedArgs.password).toString()).toString();
+var encryptedMessage = hmac + encrypted;
+
+var data = {
+    title: namedArgs.title != null ? namedArgs.title : "Protected Page",
+    instructions: namedArgs.instructions != null ? namedArgs.instructions : "",
+    encrypted: encryptedMessage,
+    crypto_tag: SCRIPT_TAG,
+    embed: namedArgs.embed != null ? namedArgs.embed : false,
+    outputFilePath: namedArgs.output != null ? namedArgs.output : namedArgs.input.replace('.html', '') + "_encrypted.html"
```

Comment:
It's probably never going to pause any issue in the real world but if our filename is `foo.html_bar.html` our final name would be `foo_bar.html_encrypted.html`.

We can use `namedArgs.input.replace(/\.html$/, '')` to only catch `.html` if it's at the extension